### ISSUE TITLE
Translate Python 2 to Python 3 code

### ISSUE THREAD

**Issue by blackHatMonkey** [sophron/wifiphisher#426]
Since Python 2 has been deprecated I believe we should move all the code to Python 3.

**Comment by mcocigra** [comment]
But also we should keep a absolutely bug free version that works and that should not be touched    so we can access a version without problems when we need.

**Comment by laozi999** [comment]
@mcocigra that is already that case if you have installed all the dependencies.

**Comment by sophron** [comment]
I don't think this is something that we should worry about now. We have <a href="https://pythonclock.org/">plenty of time</a>.

**Comment by llazzaro** [comment]
I started to migrate some code. I find hard time to make work dbus and python3.
I never used dbus before, do you think it can be replaced?

**Comment by blackHatMonkey** [comment]
@llazzaro Based on the information [here](https://pypi.python.org/pypi/dbus-python/1.2.4) it does support `python3` as well.

**Comment by llazzaro** [comment]
I know. I compiled it from source, but I got import error on SystemBus.
On site-packages the SystemBus class exists, but for some reason import is broken with that version of dbus.

**Comment by llazzaro** [comment]
I know. I compiled it from source, but I got import error on SystemBus.
On site-packages the SystemBus class exists, but for some reason import is broken with that version of dbus.

**Comment by llazzaro** [comment]
I compiled it from source, but I got import error on SystemBus.
On site-packages the SystemBus class exists, but for some reason import is broken with dbus and python3.

**Comment by sophron** [comment]
Set this as a milestone for v2.0.

**Comment by Hritik14** [comment]
I suppose it should at least be mentioned on the README that it requires python2 as now python 3 is the standard.

**Comment by guiming159** [comment]
Yes, we should mention that python 2 is needed. I made a mistake when i was installing the software.
